R/retrieveData.R

```diff
@@ -170,9 +174,9 @@ retrieveData <- function(model, rev = 0, dev = "", cachetype = "rev", ...) {
     vcat(2, " - function ", functionname, " finished", fill = 300, show_prefix = FALSE)
 
     cwd <- getwd()
-    setwd(sourcefolder)
+    setwd(sourcefolder) # nolint
     system(paste0("tar --create --gzip --file ../", collectionname, ".tgz", " ./*"))
```

Comment:
If we do this we don't need the `cwd` and `setwd` stuff:
```suggestion
    withr::with_dir(sourcefolder, system(paste0("tar --create --gzip --file ../", collectionname, ".tgz", " ./*")))
```

Reply:
done